test/cmdlineTests.sh

```diff
@@ -372,6 +370,41 @@ function test_via_ir_equivalence()
     rm -r "$SOLTMPDIR"
 }
 
+function test_cli_and_standard_json_equivalence
+{
+    (( $# == 5 )) || assertFail
+    local cli_options="$1"
+    local selected_cli_output="$2"
+    local standard_json_settings="$3"
+    local selected_standard_json_output="$4"
+    local input_file_relative_path="$5"
+
+    # CLI normalizes paths, Standard JSON uses them as is. To avoid differences, we want a path
+    # that won't change due to normalization. Only a basic sanity check not covering all cases:
+    [[ "$input_file_relative_path" =~ ^/ ]] && assertfail
+    [[ "$input_file_relative_path" =~ ^\. ]] && assertfail
```

Comment:
Do you want to match anything starting with `\`(backslash) or just `\.something`? Because I guess this expression with match only the later (the one that has the dot after the backslash), i.e. `\something` will not match and pass this test.

Reply:
Oh wait, I guess you want to match hidden directories with the second regex, right? So in that case it is correct indeed. But we could simplify the regex using the suggestion in the other comment. Or there are some corner cases that you noticed?

Reply:
I actually wanted to match things like `.`, `..`, `./x` or `../a/b/c`. I.e. I want it to reject relative paths that don't start with a file or subdirectory in the current directory. `solc` normalizes paths you give it on the CLI and it may turn such non-canonical paths into something else.

But I did not consider hidden files starting with a dot :) Overall this was meant to be only a very quick and simple sanity check. I'll need to rework it though because you make a good point.

Reply:
Done. I changed the regex to `^/|^\.$|\./|^\.\.$|\.\./|//` to make it more comprehensive and also added more info in the comment to make it clear what I'm checking for.

Reply:
Ah right, I understand now. You could simplify it a bit more to the following:
```
    [[ $input_file_relative_path =~ ^/|^\.$|\./|\.\.|// ]] && assertfail
```
Since `..` anywhere in the path should be invalid, right?

Reply:
Well, technically something like `..xyz` is still a valid name. I mean, my regex is not perfect either because it will reject `xyz.` or `xyz..`. I think I'll keep the one I have now simply because it's more regular and to avoid messing with it more. It's good enough IMO and if we hit any of such cases we'll adjust it :)